src/GovUk.Education.ExploreEducationStatistics.Content.Model.Tests/Repository/MethodologyRepositoryTests.cs

```diff
@@ -250,12 +251,247 @@ await using (var contentDbContext = InMemoryContentDbContext(contentDbContextId)
             MockUtils.VerifyAllMocks(methodologyParentRepository);
         }
 
+        [Fact]
+        public async Task GetLatestPublishedByMethodologyParent()
+        {
+            var previousVersion = new Methodology
+            {
+                Id = Guid.Parse("7a2179a3-16a2-4eff-9be4-5a281d901213"),
+                PreviousVersionId = null,
+                PublishingStrategy = Immediately,
+                Status = Approved,
+                Version = 0
+            };
+
+            var latestPublishedVersion = new Methodology
```

Comment:
Not a biggie at all, but I personally prefer to use variables here where I can e.g. `PreviousVersionId = previousVersion.Id` rather than needing to keep a track of hard-coded Guids

Reply:
Thanks, I've made this change now.